demos/react-jsnation-workshop/src/components/steps/Step6.tsx

```diff
@@ -173,12 +173,12 @@ function Profile({ user }: { user: LocalUser | undefined }): JSX.Element {
 }
 
 class LocalUser {
-  #id: number;
```

Comment:
why get rid of private fields?

Reply:
I added a lint to warn about unused private fields (because they're basically local variables).